packages/component-submission/client/graphql/withGQL.js

```diff
@@ -17,7 +17,7 @@ export default compose(
     options: props => ({ variables: { id: props.match.params.id } }),
   }),
   graphql(UPDATE_MANUSCRIPT, {
-    name: 'updateManuscript',
+    name: 'updateSubmission',
```

Comment:
Where does `UPDATE_MANUSCRIPT` come from on the line above?

Reply:
Good catch, this import doesn't exist anymore, needs changing on line 6